zenodo/modules/pages/templates/zenodo_pages/contact_form.html

```diff
@@ -80,6 +80,8 @@
 {%- endblock page_body %}
 
 {%- block javascript %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
```

Comment:
Done and thanks. We need to include zenodo.js explicitly using `{% assets "zenodo_theme_js" %}<script src="{{ ASSET_URL }}"></script>{% endassets %}` .